packages/react/src/components/DataGrid/index.tsx

```diff
@@ -41,6 +41,7 @@ export type Column<T extends Row = Row> = (
   headerName?: string;
   highlight?: boolean;
   enableSorting?: boolean;
+  width?: number;
```

Comment:
I wonder if this value should not be a string or a number.
If this is a number, we set the value as pixel. But through string, developer will be able to set width with custom units like `vw` ?

Reply:
Ok ! Maybe we could align with those props ? WDYT

Reply:
Agreed!